scripts/export/export_to_trt_llm.py

```diff
@@ -14,33 +14,21 @@
 
 import argparse
 import logging
-import sys
```

Comment:
There are mostly cosmetic or simple updates here. New addition is `--use_nim_format` flag